packages/prompt/package.json

```
@@ -0,0 +1,82 @@
+{
+	"name": "@sapphire/plugin-prompt",
+	"version": "0.0.1",
+	"description": "Plugin for @sapphire/framework to support message prompts.",
+	"author": "@sapphire",
+	"license": "MIT",
+	"main": "dist/index.js",
+	"module": "dist/index.mjs",
+	"types": "dist/index.d.ts",
+	"exports": {
+		".": {
+			"import": "./dist/index.mjs",
+			"require": "./dist/index.js"
+		},
+		"./register": {
+			"import": "./register.mjs",
+			"require": "./register.js"
+		},
+		"./register-discordjs": {
+			"import": "./register-discordjs.mjs",
+			"require": "./register-discordjs.js"
+		}
+	},
+	"homepage": "https://github.com/sapphire-project/plugins/tree/main/packages/prompt",
+	"scripts": {
+		"test": "jest",
+		"lint": "eslint src tests --ext ts --fix",
+		"build": "tsc -b src",
+		"postbuild": "run-p esm:**",
+		"esm:register": "gen-esm-wrapper dist/register.js dist/register.mjs",
+		"esm:register-discordjs": "gen-esm-wrapper dist/register-discordjs.js dist/register-discordjs.mjs",
+		"esm:default": "gen-esm-wrapper dist/index.js dist/index.mjs",
+		"prepublishOnly": "yarn build"
+	},
+	"dependencies": {
+		"@sapphire/utilities": "^1.3.0"
```

```suggestion
		"@sapphire/utilities": "^1.3.0",
		"tslib": "^2.0.1"
```

because it's required for register

---

Hmm, my IDE was complaining about this. But it compiled fine so I figured it might've been my IDE complaining for nothing as i18n didn't have it either.